app/src/hooks/useDataLoader.test.tsx

```diff
@@ -0,0 +1,189 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import { Deferred } from 'test-helpers/promises';
+import useDataLoader from './useDataLoader';
```

Comment:
This file just needs a format-fix to adjust formatting.

Reply:
Woops 😬